skin/src/main/java/org/researchstack/skin/ui/SplashActivity.java

```diff
@@ -30,14 +30,14 @@ public void onDataReady()
                 .initialize(this)
                 .compose(ObservableUtils.applyDefault())
                 .subscribe(response -> {
+                    boolean boarded = AppPrefs.getInstance(this).isOnboardingComplete();
+                    boolean signedin = DataProvider.getInstance().isSignedIn(this);
+                    boolean emailverified = AppPrefs.getInstance(this).isEmailVerified();
 
-                    if(AppPrefs.getInstance(this).isOnboardingComplete() ||
```

Comment:
I can´t remember now exactly (too long ago!), but basically, without this change the onboarding was buggy. If one stopped the app in the middle of the process the app went into some incoherent state and could not be restored.